README.md

```diff
@@ -4,7 +4,11 @@ count files, grouped by extension, in a directory. By default, it will count
 files recursively in current working directory and all of its subdirectories,
 and will display a table showing the frequency for each file extension (e.g.:
 .txt, .py, .html, .css) and the total number of files found. Any hidden files
-or folders (those with names starting with '.') are ignored by default.
+or folders are ignored by default.
+
+Supported operating systems: Linux, Mac OS, Windows.
+
+[README in Russian](https://github.com/victordomingos/Count-files/blob/master/docs/README_RU.md)
```

Comment:
I suggest to change this to:

```README in other languages: [Russian (https://github.com/victordomingos/Count-files/blob/master/docs/README_RU.md)```

So that we can add other languages later (I plan to do the Portuguese translation soon).